src/hardware/innovation.cpp

```diff
@@ -195,7 +196,7 @@ void Innovation::Render()
 		while (n < SAMPLES_PER_BUFFER) {
 			const auto buffer_pos = buffer.data() + n;
 			const auto n_remaining = SAMPLES_PER_BUFFER - n;
-			const auto cycles = cycles_per_sample * n_remaining;
+			const auto cycles = static_cast<unsigned>(cycles_per_sample * n_remaining);
```

Comment:
`service->clock()` takes an `unsigned int` for `cycles`, do we want `static_cast<unsigned int>` instead?

Reply:
Added; thanks @kklobe.